jonprl-mode.el

```diff
@@ -48,6 +48,22 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   ("<>" . ?★)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   ("base" . ?𝔻)
```

Comment:
D for domain in the sense of Scott. I wasn't sure if that terminology was used by the Nuprl folks but it seemed to fit and it is mentioned in some places like [here](http://www.cs.cornell.edu/~rahli/articles/slides-itp2013.pdf).